### PR TITLE
new queue names

### PR DESCRIPTION
**Pull request by LukeLoze1**
# Motivation and Context
Consistent Queue naming

# What has changed
The names of queues 

# How to test?
mvn clean install

# Links
https://trello.com/c/u6RN0Fuo/2372-in-ssdc-fix-queue-names-now-to-be-consistent-5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ Example Msg:
 ```
 
 ### Survey Launched
-queue: survey.launched
+queue: events.caseProceessor.surveyLaunched
```
Comment (NickGrantONS) [inline README.md:119]:
```suggestion
queue: events.caseProcessor.surveyLaunched
```

File: pom.xml [hunk 2]
```diff
@@ -176,7 +176,7 @@
             </goals>
             <phase>package</phase>
             <configuration>
-              <repository>eu.gcr.io/ons-ci-rm/rm/${project.artifactId}</repository>
+              <repository>eu.gcr.io/ssdc-rm-ci/rm/${project.artifactId}</repository>
```
Comment (NickGrantONS) [inline pom.xml:179]: merge from main and this will disappear

File: src/main/java/uk/gov/ons/ssdc/caseprocessor/service/CaseService.java [hunk 3]
```diff
@@ -16,7 +16,7 @@
 
 @Service
 public class CaseService {
-  public static final String CASE_UPDATE_ROUTING_KEY = "event.case.update";
+  public static final String CASE_UPDATE_ROUTING_KEY = "caseProcessor.rh.case";
```
Comment (NickGrantONS) [inline src/main/java/uk/gov/ons/ssdc/caseprocessor/service/CaseService.java:19]: case what?

Comment (NickGrantONS) [inline src/main/java/uk/gov/ons/ssdc/caseprocessor/service/CaseService.java:19]:
```suggestion
  public static final String CASE_UPDATE_ROUTING_KEY = "caseProcessor.rh.caseUpdate";
```

File: src/main/java/uk/gov/ons/ssdc/caseprocessor/service/UacService.java [hunk 4]
```diff
@@ -16,7 +16,7 @@
 
 @Service
 public class UacService {
-  private static final String UAC_UPDATE_ROUTING_KEY = "event.uac.update";
+  private static final String UAC_UPDATE_ROUTING_KEY = "caseProcessor.rh.uac";
```
Comment (NickGrantONS) [inline src/main/java/uk/gov/ons/ssdc/caseprocessor/service/UacService.java:19]: uac what?

Comment (NickGrantONS) [inline src/main/java/uk/gov/ons/ssdc/caseprocessor/service/UacService.java:19]:
```suggestion
  private static final String UAC_UPDATE_ROUTING_KEY = "caseProcessor.rh.uacUpdate";
```

File: src/main/resources/application.yml [hunk 5]
```diff
@@ -31,14 +31,16 @@ spring:
...
-  sample-queue: case.sample.inbound
-  receipt-response-queue: Case.Responses
-  refusal-response-queue: case.refusals
-  print-queue: Action.Printer
-  invalid-address-queue: case.invalidaddress
-  survey-launched-queue: survey.launched
+  sample-queue: supportTool.caseProcessor.sample
+  receipt-response-queue: events.caseProcessor.responses
+  refusal-response-queue: events.caseProcessor.refusals
+  print-queue: caseProcessor.printFileSvc.printBatch
```
Comment (NickGrantONS) [inline src/main/resources/application.yml:41]:
```suggestion
  print-queue: caseProcessor.printFileSvc.printBatchRow
```

File: src/main/resources/application.yml [hunk 6]
```diff
@@ -31,14 +31,16 @@ spring:
...
   telephone-capture-queue: caseApi.caseProcessor.telephoneCapture
-  sample-queue: case.sample.inbound
-  receipt-response-queue: Case.Responses
-  refusal-response-queue: case.refusals
-  print-queue: Action.Printer
-  invalid-address-queue: case.invalidaddress
-  survey-launched-queue: survey.launched
+  sample-queue: supportTool.caseProcessor.sample
+  receipt-response-queue: events.caseProcessor.responses
+  refusal-response-queue: events.caseProcessor.refusals
```
Comment (NickGrantONS) [inline src/main/resources/application.yml:40]:
```suggestion
  refusal-response-queue: events.caseProcessor.refusal
```

File: src/main/resources/application.yml [hunk 7]
```diff
@@ -31,14 +31,16 @@ spring:
...
 queueconfig:
   telephone-capture-queue: caseApi.caseProcessor.telephoneCapture
-  sample-queue: case.sample.inbound
-  receipt-response-queue: Case.Responses
-  refusal-response-queue: case.refusals
-  print-queue: Action.Printer
-  invalid-address-queue: case.invalidaddress
-  survey-launched-queue: survey.launched
+  sample-queue: supportTool.caseProcessor.sample
+  receipt-response-queue: events.caseProcessor.responses
```
Comment (NickGrantONS) [inline src/main/resources/application.yml:39]:
```suggestion
  receipt-response-queue: events.caseProcessor.response
```

File: src/test/java/uk/gov/ons/ssdc/caseprocessor/testutils/TestConstants.java [hunk 8]
```diff
@@ -1,6 +1,6 @@
 package uk.gov.ons.ssdc.caseprocessor.testutils;
 
 public class TestConstants {
-  public static final String OUTBOUND_UAC_QUEUE = "case.rh.uac";
-  public static final String OUTBOUND_CASE_QUEUE = "case.rh.case";
+  public static final String OUTBOUND_UAC_QUEUE = "caseProcessor.rh.uac";
```
Comment (NickGrantONS) [inline src/test/java/uk/gov/ons/ssdc/caseprocessor/testutils/TestConstants.java:4]:
```suggestion
  public static final String OUTBOUND_UAC_QUEUE = "caseProcessor.rh.uacUpdate";
```

File: src/test/java/uk/gov/ons/ssdc/caseprocessor/testutils/TestConstants.java [hunk 9]
```diff
@@ -1,6 +1,6 @@
...
+  public static final String OUTBOUND_CASE_QUEUE = "caseProcessor.rh.case";
```
Comment (NickGrantONS) [inline src/test/java/uk/gov/ons/ssdc/caseprocessor/testutils/TestConstants.java:5]:
```suggestion
  public static final String OUTBOUND_CASE_QUEUE = "caseProcessor.rh.caseUpdate";
```

**Review by NickGrantONS — CHANGES_REQUESTED** [review]
The only thing that really bothers me is the typo, but.

I would prefer:

- Topic should be more descriptive (i.e. uacUpdate instead of uac)
- Topic should be consistently singular (i.e. addressInvalid instead of responses/invalids/ASDAs/tescos/sainsburysssses)
- This is the last chance to get the standards good and consistent... so let's start as we mean to go on

**Comment by NickGrantONS** [comment]
update your branch from main please... cant review this as is

**Review by NickGrantONS — COMMENTED** [review]
Please update from main branch

**Review by NickGrantONS — APPROVED** [review]
Builds and ATs passing 👍

**Review by AdamHawtin — APPROVED** [review]
AT's passing locally 👍

**Pull request merged**